client/disputes/utils.ts

```diff
@@ -76,7 +76,7 @@ export const isInquiry = ( dispute: Pick< Dispute, 'status' > ): boolean => {
  * Returns the dispute fee balance transaction for a dispute if it exists
  * and the deduction has not been reversed.
  */
-export const getDisputeDeductedBalanceTransaction = (
+const getDisputeDeductedBalanceTransaction = (
```

Comment:
This is now no longer exported since its only use outside of this file has now been removed.